### DISCUSSION TITLE
Update problem

### DISCUSSION THREAD

**Discussion by BlackSparkz** [Shabinder/SpotiFlyer#3034] [Q&A]
How to update this application to 4.0.0
I'm on 3.6.3 where is update download link

**Comment by Shabinder** [comment]
#3013 

Installation Guide:
https://soundbound.shabinder.in/install
View video in above for a tutorial.

**Reply by BlackSparkz** [reply]
Not soundbond I'm discussing about Spotiflyer how to update

**Reply by Shabinder** [reply]
You cant, I have discontinued further development of spotiflyer in favor of soundbound.

**Reply by BlackSparkz** [reply]
But it's showing an update current version 3.6.3 to update version 4.0.0

**Reply by Shabinder** [reply]
Yes thats pointing to soundbound only, spotiflyer was discontinued and all future efforts from my side will be made on soundbound.

**Reply by BlackSparkz** [reply]
Ooo mean seamless promotion 😁😁
That's okay got it

**Reply by Aryan2115** [reply]
But how can I install soundbound because the link of soundbound is show an error

**Reply by Electron-webb** [reply]
Every time I search for a song on soundbound and it show and then I click on it it keeps loading and loading

**Comment by SAPPHIRE2034X** [comment]
SPOTIFLYER WAS THE PLACE FOR ME TO DOWNLOAD SONGS😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭😭

**Reply by BlackSparkz** [reply]
But still working
